test/e2e/storage/vsphere/nodemapper.go

```diff
@@ -33,9 +33,11 @@ import (
 	neturl "net/url"
 )
 
+// NodeMapper holds method for node info map
```

Comment:
nit: I think this comment could explain a bit more about what `NodeMapper` does. Maybe something like, `NodeMapper holds detailed info about nodes and datastore zones.`

Reply:
I think this show be modified in a separate PR which move `nameToNodeInfo` and `vcToZoneDatastoresMap` into this Object.

Reply:
I figured since this is a brand new comment, we could put a little more detail into it.

Reply:
In the next PR, I will move the two objects in NodeMapper and modify this comment. Is this OK?

Reply:
Yes, that would be appreciated. :smile: